client/my-sites/email/email-provider-features/toggle-button.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'calypso/components/gridicon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+
+function EmailProviderFeaturesToggleButton( { handleClick, isSwitched } ) {
+	const translate = useTranslate();
+
+	return (
+		<Button className="email-provider-features__toggle-button" onClick={ handleClick }>
+			<span>{ translate( 'Learn more' ) }</span>
+
+			<Gridicon icon={ isSwitched ? 'chevron-up' : 'chevron-down' } />
+		</Button>
+	);
+}
+
+EmailProviderFeaturesToggleButton.propTypes = {
+	handleClick: PropTypes.func.isRequired,
+	isSwitched: PropTypes.bool.isRequired,
```

Comment:
I still dislike `isSwitched` because it has the same problem as `isToggled` for me -- the prop describes the _action_ of switching/toggling rather than the actual state of the component, and every button click involves a toggle/switch, so there is no way to infer what `<EmalProviderFeaturesToggleButton isSwitched />` means in terms of expanded/contracted state.

With that in mind, does it make sense to call this something like `isRelatedContentExpanded`? While it's verbose, I think it more accurately represents the component state. As you pointed out `isExpanded`, is inaccurate because this component doesn't expand or contract.